scripts/common.sh

```
@@ -375,29 +744,28 @@ function list_branches {
 
     if [[ "$1" == "delete" ]] && [[ "$number_of_branches" == 2 ]] && [[ "${current_branch}" != "${main_branch}" ]]; then
         echo
-        echo -e "${YELLOW}There is no branches to delete${ENDCOLOR}"
-        exit
+        echo -e "${YELLOW}There are no branches to delete${ENDCOLOR}"
+        to_exit="true"
+        return
     fi
 
     ### Main should be the first
     branches_first_main=(${main_branch})
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)